catalyst_voices/lib/widgets/modals/voices_upload_file_dialog.dart

```diff
@@ -0,0 +1,371 @@
+                },
+              ),
+              InkWell(
+                onTap: () async {
+                  final result = await FilePicker.platform.pickFiles(
+                    type: (widget.allowedExtensions != null)
+                        ? FileType.custom
+                        : FileType.any,
+                    allowedExtensions: (widget.allowedExtensions != null)
+                        ? widget.allowedExtensions!
+                        : null,
+                  );
+                  final file = result?.files.first;
+                  final name = file?.name;
+                  final bytes = file?.bytes;
+
+                  if (name != null && bytes != null) {
+                    widget.onFileSelected?.call(
+                      VoicesFile(
+                        name: name,
+                        bytes: bytes,
+                      ),
+                    );
+                  }
+                },
+                child: Container(
+                  width: double.infinity,
+                  height: double.infinity,
+                  alignment: Alignment.center,
+                  child: Wrap(
+                    direction: Axis.vertical,
+                    crossAxisAlignment: WrapCrossAlignment.center,
+                    children: [
+                      Container(
+                        width: 80,
+                        height: 80,
+                        padding: const EdgeInsets.all(20),
+                        decoration: BoxDecoration(
+                          shape: BoxShape.circle,
+                          border: Border.all(
+                            color: Theme.of(context).colors.iconsPrimary!,
+                            width: 3,
+                          ),
+                        ),
+                        child: VoicesAssets.icons.upload.buildIcon(
+                          color: Theme.of(context).colors.iconsPrimary,
+                        ),
+                      ),
+                      RichText(
+                        text: TextSpan(
+                          text: context.l10n
+                              .uploadDropInfo(widget.itemNameToUpload),
+                          style: Theme.of(context).textTheme.titleSmall,
+                          children: <TextSpan>[
+                            TextSpan(
+                              text: context.l10n.browse,
+                              style: TextStyle(
+                                color: Theme.of(context).colors.iconsPrimary,
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      if (widget.info != null)
+                        Text(
+                          widget.info!,
+                          style: Theme.of(context).textTheme.bodyMedium,
+                        ),
+                    ]
+                        .separatedBy(
+                          Container(
```

Comment:
We use SizedBox as separator because it has const constructor, it should be possible to:

```
.separatedBy(const SizedBox(height: 12))
```